packages/algob/src/builtin-tasks/test.ts

```diff
@@ -6,10 +7,17 @@ import type { Config } from "../types";
 import { loadFilenames } from "./deploy";
 import { TASK_TEST } from "./task-names";
 
+const TEST_DIR = 'test';
 async function runTests (config: Config): Promise<void> {
   try {
-    const testFiles = loadFilenames(testsDirectory, "test");
-    console.log("Test files:", testFiles);
+    const tsPath = findupSync("tsconfig.json", { cwd: process.cwd() });
+    if (tsPath) {
+      // run tests via ts-mocha, if project is in typescript
+      process.env.TS_NODE_PROJECT = tsPath;
+      require('ts-mocha');
+    }
+
+    const testFiles = loadFilenames(testsDirectory, TEST_DIR);
```

Comment:
do we have test for this ?

Reply:
if not, please add it.

Reply:
this is a common function for deploy and run tasks as well (to load scripts). We have test for it in deploy task - [https://github.com/scale-it/algo-builder/blob/master/packages/algob/test/builtin-tasks/deploy.ts#L15](https://github.com/scale-it/algo-builder/blob/master/packages/algob/test/builtin-tasks/deploy.ts#L15). I think all tests for deploy/run would fail if this function fails.
Let me know if you want to add more tests.

Reply:
I think this test is already passing. so why are we adding new feature?  if that test passes before and after that means it doesn't test the feature added here

Reply:
> do we have test for this ?

this was confusing. I thought you meant test for `loadFilenames`.

Reply:
will add test for `process.env.TS_NODE_PROJECT`. 

BDW - how do we add test for a command which runs mocha in project root (basically a test for a command which runs all tests in project) ? The feature here is to run mocha on ts-files as well.